Rogue-Ape-Studios-XR/Secrets-of-Ignacios
# 🐛 Grimoire fixes

## Content

Fixed issue with grimoire dissappearing and improved it's performance

## Changes

### Added
`Assets/Prefabs/PlayerRig/PlayerRig with fixed book.prefab` : A temporary prefab for the player rig which contains improvements and bugfixes

### Updated

`Assets/Scripts/UI/GrimoireUIController.cs`: Was updated with new code to support the new changes

## Testing

The feature / Bugfix can be testing by following these steps:

1. Enter main scene with PlayerRig with fixed book prefab
2. See if the grimoire works well, 
3. Try putting your hand through the cauldron and see if you can interact with the grimoire still


Closes #187 #173
